osu.Game/Online/API/APIAccess.cs

```diff
@@ -399,7 +399,10 @@ public void Queue(APIRequest request)
             lock (queue)
             {
                 if (state.Value == APIState.Offline)
+                {
+                    request.Fail(new WebException("Disconnected from server"));
```

Comment:
Not sure this is the best exception message, as `Offline` implies the user logged out (disconnections are `APIState.Failing` in general). Is covered in the `APIState` xmldoc, but maybe we should renamed `Offline` to avoid confusion? `LoggedOut` or something?

```suggestion
                    request.Fail(new WebException("User is not logged in"));
```

Reply:
I used the same message as `flushQueue`.

Reply:
Okay if I update both in that case? I feel like this will be read incorrectly when it comes up in logs.

Reply:
Go for it